src/rpc/blockchain.cpp

```diff
@@ -987,6 +1016,12 @@ UniValue gettxout(const JSONRPCRequest& request)
     if (request.params.size() > 2)
         fMempool = request.params[2].get_bool();
 
+    bool include_spent = false;
+    if (request.params.size() > 3) {
+        include_spent = request.params[3].get_bool();
+    }
```

Comment:
Disallow `include_spent` if `include_mempool`:
```cpp
    if (fMempool && include_spent) {
        throw JSONRPCError(RPC_INVALID_PARAMETER, "...");
    }
```
If you add this, update test below.